src/internal/typestyle.ts

```diff
@@ -31,6 +31,7 @@ export class TypeStyle {
   private _pendingRawChange: boolean;
   private _raw: string;
   private _tag?: StylesTarget;
+  private _window: Window = window;
```

Comment:
You're right, that's why I had to add jsDom for the tests, but i guess I didn't think about server side. But I can then get rid of jsDom and check for undefined